toolsay/toolsay.csproj

```diff
@@ -69,4 +71,29 @@
     <Delete Files="@(_PackageFilesToDelete)"/>
   </Target>
 
+  <Target Name="_AlsoPackTool">
+    <ItemGroup>
+      <_ToolProjectToPack Include="$(MSBuildProjectFullPath)" AdditionalProperties="PackAsTool=true" />
+    </ItemGroup>
+    <MSBuild Projects="@(_ToolProjectToPack)" Targets="PackTool">
+      <Output TaskParameter="TargetOutputs" ItemName="TfmSpecificPackageFile" />
+    </MSBuild>
+  </Target>
```

Comment:
This causes the framework-agnostic tool stuff to get added to the library package. If we're not building a framework-agnostic tool pack then this ensures that the RID-specific DotnetToolSettings.xml gets added appropriately as well. Essentially we're making the library Package also work as the 'pointer' tool package.